lib/matplotlib/backends/backend_qt5.py

```diff
@@ -493,8 +497,7 @@ def notify_axes_change(fig):
 
     @QtCore.Slot()
     def _show_message(self, s):
-        # Fixes a PySide segfault.
```

Comment:
Can we drop this helper function and just hook up `self.statusbar_label.setText` to the proper signal?